hydroflow/examples/kvs_bench/main.rs

```diff
@@ -26,18 +26,23 @@ struct Cli {
 #[derive(Debug, Subcommand)]
 enum Commands {
     Bench {
+        /// Number of threads to run on .
         #[clap(long, default_value_t = 1)]
         threads: usize,
 
+        /// `s` value for the zipf sampling distribution for keys.
         #[clap(long, default_value_t = 4.0)]
         dist: f64,
 
+        /// How long to warm up for, in seconds.
         #[clap(long, default_value_t = 2)]
         warmup: u64,
```

Comment:
Minor & non-blocking:
If we're in a situation where we can't use Duration (perhaps because it's difficult to make it work with Clap), it is nice to include the unit in the name itself as a poor man's way of making units clear everywhere. So `warmup_seconds` (or similar) instead of `warmup`.

Reply:
Also, this isn't part of your change. (I'm aware).